### PR TITLE
Run more mac CI

### PR DESCRIPTION
**Pull request by jrmuizel**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/3027)

### CODE REVIEW

**Comment by staktrace** [comment]
This is touching the linux part of the yml file, not the mac part

**Comment by jrmuizel** [comment]
Quite.

File: .taskcluster.yml [hunk 1]
```diff
@@ -153,7 +153,8 @@ tasks:
           export PKG_CONFIG_PATH="/usr/local/opt/zlib/lib/pkgconfig:$PKG_CONFIG_PATH" &&
           export RUSTC_WRAPPER=sccache &&
           (cd wrench && python script/headless.py reftest) &&
-          (cd wrench && cargo build --release)
+          (cd wrench && cargo build --release) &&
+          (cd wrench && cargo run --release -- reftest reftests/clip/fixed-position-clipping.yaml)
```
Comment (kvark) [inline .taskcluster.yml]: we should just say `cargo run --release -- --precache show reftests/blend/blank.yaml` instead

Comment (jrmuizel) [reply .taskcluster.yml]: show doesn't terminate, that's why I chose reftest.

**Pull request merged**